plasma_framework/test/src/exits/payment/PaymentUpdateBond.test.js

```diff
@@ -0,0 +1,68 @@
+const ExpectedOutputGuardHandler = artifacts.require('ExpectedOutputGuardHandler');
+const OutputGuardHandlerRegistry = artifacts.require('OutputGuardHandlerRegistry');
+const PaymentChallengeStandardExit = artifacts.require('PaymentChallengeStandardExit');
+const PaymentProcessStandardExit = artifacts.require('PaymentProcessStandardExit');
+const PaymentStandardExitRouter = artifacts.require('PaymentStandardExitRouterMock');
+const PaymentStartStandardExit = artifacts.require('PaymentStartStandardExit');
+const PaymentSpendingConditionRegistry = artifacts.require('PaymentSpendingConditionRegistry');
+const SpyPlasmaFramework = artifacts.require('SpyPlasmaFrameworkForExitGame');
+const SpyEthVault = artifacts.require('SpyEthVaultForExitGame');
+const SpyErc20Vault = artifacts.require('SpyErc20VaultForExitGame');
+
+const { expectEvent } = require('openzeppelin-test-helpers');
+
+contract('PaymentStandardExitRouter', ([_, outputOwner]) => {
+    const CHILD_BLOCK_INTERVAL = 1000;
+    const MIN_EXIT_PERIOD = 60 * 60 * 24 * 7; // 1 week
+    const DUMMY_INITIAL_IMMUNE_VAULTS_NUM = 0;
+    const INITIAL_IMMUNE_EXIT_GAME_NUM = 1;
+    const PAYMENT_OUTPUT_TYPE = 1;
+    const EMPTY_BYTES = '0x0000000000000000000000000000000000000000000000000000000000000000000000';
+
+    before('deploy and link with controller lib', async () => {
+        const startStandardExit = await PaymentStartStandardExit.new();
+        const challengeStandardExit = await PaymentChallengeStandardExit.new();
+        const processStandardExit = await PaymentProcessStandardExit.new();
+
+        await PaymentStandardExitRouter.link('PaymentStartStandardExit', startStandardExit.address);
+        await PaymentStandardExitRouter.link('PaymentChallengeStandardExit', challengeStandardExit.address);
+        await PaymentStandardExitRouter.link('PaymentProcessStandardExit', processStandardExit.address);
+    });
+
+    describe('startStandardExit', () => {
```

Comment:
```suggestion
    describe('updateStartStandardExitBondSize', () => {
```

Reply:
Also, I think we should add another tests on it would actually change the bond size (?)

```
it('should update the standard exit bond size after the waiting period passes')
```

Probably we can move
```
const newBondSize = this.startStandardExitBondSize.addn(20);
const { receipt } = await this.exitGame.updateStartStandardExitBondSize(newBondSize);
```
to beforeEach? so we re-use on different tests.

Reply:
> Also, I think we should add another tests on it would actually change the bond size (?)

That's already tested in the BondSize tests. I didn't think there was any point in duplicating it here.

Reply:
hmmm, okay I will leave this to you then. 

My thought is that it is tested in BondSize but we need to make sure in PaymentExitGame is actually calling the bondSize library as dependency. Sadly as they are strongly coupled in our code as that's the nature of using library in solidity, so the most economical way we can do is to use some happy case to show it is hitting that. But I think I am in general not too worried about this so I will leave this to your decision.

Reply:
Ok. Well, it's easy to add the test and I guess it's better to have too many tests than too few 😄